configurations/dockerFileTemplate.go

```diff
@@ -8,57 +8,54 @@ import (
 )
 
 type dockerfileTemplate struct {
-	Data *dockerfileData
+	Data stage
```

Comment:
why stage again? Didn't we get rid of this word from the config?

Reply:
Yes. Renamed it